Doc/library/argparse.rst

```diff
@@ -1233,8 +1233,8 @@ container should match the type_ specified::
    usage: doors.py [-h] {1,2,3}
    doors.py: error: argument door: invalid choice: 4 (choose from 1, 2, 3)
 
-Any container can be passed as the *choices* value, so :class:`list` objects,
-:class:`set` objects, and custom containers are all supported.
+Any sequence can be passed as the *choices* value, so :class:`list` objects,
+:class:`set` objects, and custom sequences are all supported.
```

Comment:
Sets are not sequences, so this sentence doesn't make sense any more. @rhettinger wrote on the issue that sets don't work well, so I suggest we mention tuples as a second example here instead of sets.